agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/RpConfigurationPolling.java

```diff
@@ -132,7 +134,9 @@ public void run() {
         rpConfiguration = newRpConfiguration;
       }
     } catch (IOException e) {
-      logger.error("Error occurred when polling json config file: {}", e.getMessage(), e);
+      try (MDC.MDCCloseable ignored = MsgId.CONFIGURATION_RELATED_ERROR.makeActive()) {
```

Comment:
I'm not sure I'd add this one to CONFIGURATION_RELATED_ERROR, since that bucket seems more targeted at user configuration errors